src/ui/components/results/directanswercomponent.js

```diff
@@ -87,15 +99,32 @@ export default class DirectAnswerComponent extends Component {
     return true;
   }
 
+  /**
+   * Returns the custom card that should be used for the given direct answer.
+   * First, checks user given cardOverrides for a matching override, if there are none
+   * then returns the default card.
+   * @returns {string}
+   */
+  _getCustomCard (directAnswer) {
+    const answer = directAnswer.answer || {};
+    const matchingOverrides = this._cardOverrides.filter(override => {
+      const { fieldType, fieldName, entityType } = override;
+      return directAnswer.fieldType === fieldType &&
```

Comment:
Should these be ||s rather than &&s? The spec was saying that the first rule in the array that matches the direct answer would be applied, even if a more specific rule exists

Reply:
shoot yeah I goofed here. fieldType is not yet implemented, and I assumed entityType was in the same directAnswer.answer object as fieldName. It's not, so when I was testing only fieldName would be checked. Fixed now!